src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ChangeRequestsTab.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeRequest, WorkPackage } from 'shared';
+import { useAllChangeRequests } from '../../../hooks/change-requests.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { projectWbsPipe } from '../../../utils/pipes';
+import ChangeRequestRow from '../../../components/ChangeRequestRow';
+interface ChangeRequestsTabProps {
+  workPackage: WorkPackage;
+}
+
+const ChangeRequestsTab: React.FC<ChangeRequestsTabProps> = ({ workPackage }) => {
+  const { data: changeRequests, isError: crIsError, isLoading: crIsLoading, error: crError } = useAllChangeRequests();
+
+  if (crIsLoading || !changeRequests) return <LoadingIndicator />;
+  if (crIsError) return <ErrorPage message={crError?.message} />;
+
+  const unReviewedChangeRequests = changeRequests
+    ? changeRequests
+        .filter((cr: ChangeRequest) => !cr.dateReviewed && projectWbsPipe(cr.wbsNum) === projectWbsPipe(workPackage.wbsNum))
+        .sort((a, b) => b.dateSubmitted.getTime() - a.dateSubmitted.getTime())
+    : [];
+
+  const approvedChangeRequests = changeRequests
```

Comment:
you don't need to check whether changeRequests is defined, because you're returning a loadingindicator if it's undefined above, you  know it's at worst an empty array here

Reply:
same for unReviewedCRs